plan/dag_plan_test.go

```diff
@@ -380,7 +380,7 @@ func (s *testPlanSuite) TestDAGPlanBuilderSubquery(c *C) {
 		// Test Nested sub query.
 		{
```

Comment:
add test for stream agg + index join and stream agg + merge join and stream agg + limit and stream agg + sort

Reply:
Done.